src/services/accounts/AccountsBalanceInfoService.ts

```diff
@@ -1,44 +1,84 @@
-import { BlockHash } from '@polkadot/types/interfaces';
+import { Vec } from '@polkadot/types';
+import {
+	AccountData,
+	BalanceLock,
+	BlockHash,
+} from '@polkadot/types/interfaces';
+import { BadRequest } from 'http-errors';
 import { IAccountBalanceInfo } from 'src/types/responses';
 
 import { AbstractService } from '../AbstractService';
 
 export class AccountsBalanceInfoService extends AbstractService {
 	/**
 	 * Fetch balance information for an account at a given block.
+	 * N.B. assumes all non native tokens are from ORML tokens pallet.
 	 *
 	 * @param hash `BlockHash` to make call at
 	 * @param address address of the account to get the balance info of
+	 * @param token token to get the balance info of for the user
```

Comment:
```suggestion
	 * @param token Token to get the balance info of.
```

Maybe? (and if yes, same above)